src/modules/dailyCreditsSummary/i18n/pl_PL.ts

```diff
@@ -0,0 +1,106 @@
+export default {
+    amount: 'Amount',
+    charts: {
+        income: 'Dochód',
+        expenses: 'Wydatki',
+    },
+    categories: {
+        allianceMission: {
+            regex: /^\[Alliance\]/u,
```

Comment:
Is an alliance also called alliance in the polish version?

Reply:
alliance=sojusz

Reply:
As @jxn-30 mentioned: We'll leave it and work on it after the merge